src/renderer/components/uielements/assets/assetMenu/AssetMenu.tsx

```diff
@@ -19,10 +19,21 @@ type Props = {
   searchDisable: string[]
```

Comment:
Check / extend story of `AssetMenu.storybook` to show few assets (currently it displays just one)

![Screenshot from 2020-08-27 11-34-15](https://user-images.githubusercontent.com/61792675/91423769-42487680-e859-11ea-841f-e3f5331b32bb.png)